builtin/bisect--helper.c

```diff
@@ -1113,7 +1174,7 @@ int cmd_bisect__helper(int argc, const char **argv, const char *prefix)
 
 	argc = parse_options(argc, argv, prefix, options,
 			     git_bisect_helper_usage,
-			     PARSE_OPT_KEEP_DASHDASH | PARSE_OPT_KEEP_UNKNOWN);
+			     PARSE_OPT_KEEP_DASHDASH | PARSE_OPT_KEEP_UNKNOWN | PARSE_OPT_SHELL_EVAL);
```

Comment:
That option just makes sure that `-h` will output the usage in a form that can be `eval`ed easily in a shell script...

Reply:
oh, I forgot to undo this.